src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ValidateAssemblies.cs

```diff
@@ -87,6 +88,22 @@ internal static class ValidateAssemblies
             // Execute the enqueued work item(s).
             apiCompatRunner.ExecuteWorkItems();
 
+            if (serviceProvider.CompatibilityLogger.SuppressionWasLogged)
+            {
+                if (!generateSuppressionFile)
+                {
+                    serviceProvider.CompatibilityLogger.LogMessage(
+                        MessageImportance.High,
+                        CommonResources.BreakingChangesFound);
+                }
+            }
+            else
+            {
+                serviceProvider.CompatibilityLogger.LogMessage(
+                    MessageImportance.Normal,
+                    CommonResources.NoBreakingChangesFound);
+            }
```

Comment:
Should we move this into a helper function similar to the below `GenerateSuppressionFile` helper function to avoid the repetition between the two tasks?